### PR TITLE
Feature/update aws to include war

### PR DESCRIPTION
**Pull request by Amy1612**
I wasn't able to properly test the new way of enabling HTTPS, would one of you please be able to or be able to show me how to?

### CODE REVIEW

File: _includes/cloud-docker/aws.md [hunk 1]
```diff
@@ -2,146 +2,96 @@
...
+![12](/assets/images/aws/12.png)
 
-Give it a few minutes to settle. You should now be able to access the service using the Load Balancer's address, rather than relying on the IP address. To find out what it is, scroll the left side navigation bar until you see the 'Load Balancing' section, and click on 'Load Balancers'.
-![21](/assets/images/aws/21.png)
+Go back to your EC2 instance, click on 'Connect' then go to the 'SSH client' tab and copy the public DNS:
+![13](/assets/images/aws/13.png)
 
+Open a terminal and copy in this code, replacing with your information(the default username here is 'ec2-user', unless the AMI Owner has changed it):
+```
+scp -i <path/to/key.pem> path/to/war{{page.product |downcase}}-microservice.war ec2-user@<YourPublicDNS>:~
```
Comment (daniel-warren) [inline _includes/cloud-docker/aws.md:53]: Missing a /
```suggestion
scp -i <path/to/key.pem> path/to/war/{{page.product | downcase}}-microservice.war ec2-user@<YourPublicDNS>:~
```

File: _includes/cloud-docker/aws.md [hunk 2]
```diff
@@ -2,146 +2,96 @@
...
-In the description, you can see its DNS name. Click the copy icon next to it, and open it in a new tab. You should be prompted for your username and password, and then see the {{ page.product }} welcome page.
+SSH into the instance:
+```
+ ssh -i <path/to/key.pem> ec2-user@<YourPublicDNS>
+```
 
-You can now use this load balancer address to set up a DNS CNAME record for your own domain. Eg: {{ page.product | downcase }}.\<your_domain\>.com. The steps for this vary between DNS providers, but here is the new entry in Cloudflare: 
+Once in, run the docker image:
+```
+docker run -p 80:80 --mount "source=/home/ec2-user/{{page.product | dowcase}}-microservice.war,target=/usr/local/tomcat/webapps/ROOT.war,type=bind" idrsolutions/{{page.product}}
```
Comment (daniel-warren) [inline _includes/cloud-docker/aws.md:63]: Typo stops it from being changed to lowercase
```suggestion
docker run -p 80:80 --mount "source=/home/ec2-user/{{page.product | downcase}}-microservice.war,target=/usr/local/tomcat/webapps/ROOT.war,type=bind" idrsolutions/{{page.product}}
```

File: _includes/cloud-docker/aws.md [hunk 3]
```diff
@@ -2,146 +2,96 @@
...
-
-
-Click 'Save'. You should not be able to access {{ page.product }} on `https://<your.domain.name>`.
-
+To enable HTTPS, you need to open up the HTTPS port in the firewall, then you need a valid SSL certificate and key mounted at /opt/ssl in the virtual machine.
+Go to your EC2 instance and Go to the 'Security' tab and click on the 'Security groups' id.
+![15](/assets/images/aws/10.png) 
+ 
+On this page click the "Edit inbound rules" button.
+![16](/assets/images/aws/15.png)
```
Comment (daniel-warren) [inline _includes/cloud-docker/aws.md:76]: Was image 15 meant to be deleted in the commits? It's not loading for me as the file is removed

**Review by daniel-warren — CHANGES_REQUESTED** [review]
@Amy1612 Have noticed a few minor things to amend, otherwise looks promising.

File: _includes/cloud-docker/aws.md (hunk 3 above)
Comment (Amy1612) [reply _includes/cloud-docker/aws.md:76]: Sorry I didn't add the new image 15, have done so now

**Review by daniel-warren — APPROVED** [review]

**Review by KieranFrance — APPROVED** [review]
Aproved but as we are moving the site I will close this pull request and manually move changes into the new rep

**Pull request closed without merging**